advanced_functionality/multi_model_tensorflow/tf2-MME-regression.ipynb

```diff
@@ -0,0 +1,1534 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    petrolDF = pd.read_csv("petrol_consumption.csv")
</pre><p>Don't commit datasets directly to the repository. I will work on hosting it in S3, and then it can be downloaded from within the notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2887/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>